experimenter/pyproject.toml

```diff
@@ -74,6 +68,12 @@ django-filter = "^23.5"
 django-bootstrap5 = "^24.3"
 djlint = "^1.36.1"
 pyyaml = "^6.0.1"
+pillow = "^11.1.0"
```

Comment:
id love it if we sorted these deps

Reply:
+1 to that

Reply:
I know I was thinking that too but didn't wanna make noise in this pr we can do it separately.